pkg/mod/mod.go

```diff
@@ -0,0 +1,40 @@
+package mod
+
+import (
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+type goModule struct {
```

Comment:
that fact we are using `go mod` is an implementation detail, users of pkg/mod shouldnt care what type of mod it is, as long as it provides .sysl files. this could just be `module` or similar